projects/core/src/cart/store/actions/multi-cart.action.ts

```diff
@@ -41,37 +41,35 @@ export const CART_PROCESSES_INCREMENT = '[Multi Cart] Cart Processes Increment';
 export const CART_PROCESSES_DECREMENT = '[Multi Cart] Cart Processes Decrement';
 
 /**
- * To keep track of cart creation process we use cart with `fresh` id.
+ * To keep track of cart creation process we use cart with `fresh-${uuid}` id.
  * After creating cart we switch to entity with `code` or `guid`.
- * We need `fresh` cart entity for loading/error state.
+ * We need `fresh-${uuid}` cart entities for loading/error state.
  */
-export const FRESH_CART_ID = 'fresh';
-
-export class ResetFreshCart extends EntityProcessesLoaderResetAction {
+export class ResetFreshCart extends EntityRemoveAction {
```

Comment:
Can we rename the action to from `Reset...` to `Remove...` then?

Reply:
Renamed